tests/test_argparsing.py

```diff
@@ -46,16 +34,35 @@ def test_command_override_static():
     assert exp.ARGS['model']['classifier_type'] == expected_type
 
 
-def test_static_override_parameters():
-    expected_type = 'resnet'
-    classifier_resnet = ClassifierModified()
+def test_static_override_parameters(args, classifier_modified):
+    """
+
+    Args:
+        args: Namespace
```

Comment:
Please add a comment here and anywhere else that you are using the args from fixture.

Reply:
Done in commit 41921c1.